src/style.css

```diff
@@ -1,3 +1,85 @@
+:root {
+  --blue-color: #00acdc;
+}
+
+* {
+  box-sizing: border-box;
+  margin: 0;
+  padding: 0;
+}
```

Comment:
The use of the universal selector `*` is discouraged as it can lead to performance issues. Consider applying styles more selectively to improve performance.